core/src/main/java/io/grpc/util/OutlierDetectionLoadBalancer.java

```diff
@@ -89,7 +89,7 @@ public OutlierDetectionLoadBalancer(Helper helper, TimeProvider timeProvider) {
   }
 
   @Override
-  public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
```

Comment:
You're replacing the definition for handleResolvedAddresses rather than just implementing the new method. You should probably instead just have handleResolvedAddresses call acceptResolvedAddresses.  Otherwise, the old method will change to the base classes implementation causing inconsistency for anything using it.

Reply:
Okay, thanks for the explanation.